manifests/cinder-csi-plugin/cinder-csi-nodeplugin.yaml

```diff
@@ -19,10 +19,20 @@ spec:
       hostNetwork: true
       containers:
         - name: node-driver-registrar
-          image: quay.io/k8scsi/csi-node-driver-registrar:v1.2.0
+          image: quay.io/k8scsi/csi-node-driver-registrar:v2.0.1
           args:
             - "--csi-address=$(ADDRESS)"
             - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
+            - "--health-port=9809"
```

Comment:
I see that this feature is disabled by default ? Im not sure why though (need to do more reading) . Isnt it better to leave disabled and let the user enable if required?

Reply:
um.. yes, seems disable is reasonable 
thanks